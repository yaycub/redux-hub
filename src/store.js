import { createStore } from 'redux';
import thunk from 'redux-thunk';



