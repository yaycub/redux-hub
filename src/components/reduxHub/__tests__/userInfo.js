import React from 'react';
import { mount } from 'enzyme';
import UserInfo from '../userInfo';
import { Provider } from 'react-redux';
import store from '../../../store';

describe('User Info Component', () => {
  it('should render userInfo', () => {
    const wrapper = mount(
      <Provider store={store}>
        <UserInfo />
      </Provider>
    );

    expect(wrapper).toMatchSnapshot();
  });
});

