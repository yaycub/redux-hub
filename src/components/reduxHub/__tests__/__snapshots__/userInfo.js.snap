// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Info Component should render userInfo 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <UserInfo>
    <p>
      User info will display here
    </p>
  </UserInfo>
</Provider>
`;
